account_invoice_payment_ref_mandatory/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# © 2022 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+{
+    "name": "Account Invoice Payment Ref Mandatory",
+    "summary": """
+         Adds required payment_reference field
+    """,
+    "author": "Solvos",
+    "license": "AGPL-3",
```

Comment:
All OK, but this LICENSE should be changed.
```suggestion
    "license": "LGPL-3",
```

Reply:
Done 👌
thanks for the review